### PR TITLE
restyle footer

### PR DESCRIPTION
**Pull request by DanielW1987**

### CODE REVIEW

**Comment by DanielW1987** [comment]
[Add status monitor to website](https://trello.com/c/oq9GvcIH/331-add-status-monitor-to-website)

**Comment by DanielW1987** [comment]
[Donation link](https://trello.com/c/0mGgbROc/294-donation-link)

File: webapp/src/main/resources/templates/frontend/fragments/layout.html [hunk 1]
```diff
@@ -87,18 +87,57 @@
...
... 796.352 464.617 801.794 463.567 823.25 459.386 843.761 452.245 859.023 435.916c24.295-25.998 29.13-59.895 20.544-94.067zM72.4301 365.835C72.757 365.68 72.1548 368.484 71.8967 369.792 71.8451 367.813 71.9483 366.058 72.4301 365.835zM74.5121 381.94C74.6842 381.819 75.2003 382.508 75.7337 383.334 74.925 382.576 74.4089 382.009 74.4949 381.94H74.5121zM76.5597 384.641C77.2996 385.897 77.6953 386.689 76.5597 384.641v0zM80.672 387.979H80.7752C80.7752 388.1 80.9645 388.22 81.0333 388.341 80.9192 388.208 80.7925 388.087 80.6548 387.979H80.672zM800.796 382.989C793.088 390.319 781.473 393.726 769.996 395.43c-128.704 19.099-259.283 28.769-389.399 24.502C287.476 416.749 195.336 406.407 103.144 393.382 94.1102 392.109 84.3197 390.457 78.1082 383.798c-11.7004-12.561-5.9534-37.854-2.9079-53.03C77.9878 316.865 83.3218 298.334 99.8572 296.355 125.667 293.327 155.64 304.218 181.175 308.09 211.917 312.781 242.774 316.538 273.745 319.36 405.925 331.405 540.325 329.529 671.92 311.91 695.906 308.686 719.805 304.941 743.619 300.674 764.835 296.871 788.356 289.731 801.175 311.703 809.967 326.673 811.137 346.701 809.778 363.615 809.359 370.984 806.139 377.915 800.779 382.989H800.796z" fill="currentcolor"/>
```
Comment (hedgehog1833) [inline webapp/src/main/resources/templates/frontend/fragments/layout.html:110]: what's all that? oO

Comment (DanielW1987) [reply webapp/src/main/resources/templates/frontend/fragments/layout.html]: I have now decided to use simple PNGs instead of SVGs.

Auswahl_141

**Review by hedgehog1833 — APPROVED** [review]

**Pull request merged**